runtests.py

```diff
@@ -423,6 +423,7 @@ def build_extension(self, ext):
                 compiler_obj = self.compiler
             if ext.language == 'c++':
                 compiler_obj.compiler_so.remove('-Wstrict-prototypes')
+                compiler_obj.compiler_so.append('-std=c++11')
```

Comment:
We don't want to change all our tests to require this. Instead, put it as a directive in the file in question.